### PR TITLE
feat: add unmanaged support

### PR DESCRIPTION
**Pull request by danlucian**

### CODE REVIEW

File: pkg/sbom/sbom.go [hunk 1]
```diff
@@ -75,6 +76,7 @@ func Init(e workflow.Engine) error {
 	flagset := pflag.NewFlagSet("snyk-cli-extension-sbom", pflag.ExitOnError)
 
 	flagset.Bool(flagExperimental, false, "Explicitly enable `sbom` command with the --experimental flag.")
+	flagset.Bool(flagUnmanaged, false, "Use to invoke an unmanaged test")
```
[the remark this answers is not in the document]
Comment (danlucian) [reply pkg/sbom/sbom.go:79]: That's a good point, we need to change it. This is not even a `test` but a `generation`. Thank you for raising it 🙏

[the remark this answers is not in the document]
Comment (danlucian) [reply pkg/sbom/sbom.go]: Updated to 'For C++ only, detect local unmanaged open source dependencies and build an SBOM.'

[the remark this answers is not in the document]
Comment (danielroymoore) [reply pkg/sbom/sbom.go]: To clarify, is it C++ only? Is it not C / C++? Should we not be specific?

When runing the current cli help we get:


```
Note: Use snyk test --unmanaged to scan all files for known open source dependencies (C/C++
    only).
```

So my suggestion on wording would be:

"scan all files for known open source dependencies and generate an SBOM (C/C++ only)"

**Review by mcombuechen — APPROVED** [review]

File: pkg/sbom/sbom.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (danlucian) [reply pkg/sbom/sbom.go]: Thank you, Dan, indeed it should be both C and C++.

**Review by danielroymoore — APPROVED** [review]

**Review by mcombuechen — APPROVED** [review]

**Pull request merged**